src/lib/src/services/validation-message/validation-message.service.ts

```diff
@@ -46,6 +46,7 @@ export class TsValidationMessageService {
       noResults: `No results found.`,
       url: `'${validatorValue.actual}' is not a valid URL.`,
       equalToControl: `'${validatorValue.actual}' is not equal to '${validatorValue.compareValue}'`,
+      lowercase: `Must contain at least ${validatorValue.contains} lower case letters`,
```

Comment:
It looks like both `lower case` and `lowercase` are technically valid spellings. But for consistencies sake, we should change this to `lowercase` to match the phrasing elsewhere in the codebase.